### PR TITLE
Add endpoint to run GWLF-E if passed a full input dataset.

### PR DESCRIPTION
**Pull request by jtarricone**
This implements an endpoint that, when given a data dictionary, runs GWLF-E.

A starting point for testing might be creating a project/scenario and then running the "Watershed Multi-Year" model, but it could also be tested by simply posting the results of the mapshed API call to the GWLF-E one.

Connects #1297

### CODE REVIEW

**Comment by rajadain** [comment]
+1 this is working correctly as far as the scope of this card is concerned. However, due to limitations in GWLF-E, I'm getting this error:

```
[2016-05-18 13:43:41,899: ERROR/Worker-4] Task d19060f2-c5af-4fcd-9244-234a0c8fde69 run from job 2821 raised exception: 'DataModel' object has no attribute 'AnnDayHrs'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/app/apps/modeling/tasks.py", line 337, in run_gwlfe
    response_json = {'runoff': gwlfe.run(z)}
  File "/opt/app/requirements/src/gwlf-e/gwlfe/gwlfe.py", line 38, in run
    ReadGwlfDataFile.ReadAllData(z)
  File "/opt/app/requirements/src/gwlf-e/gwlfe/ReadGwlfDataFile.py", line 30, in ReadAllData
    z.AnnDayHrs += z.DayHrs[i]
AttributeError: 'DataModel' object has no attribute 'AnnDayHrs'
```

I'm working out the details, but I almost wonder if we should have an option that returns the values of the sample file if say `model_input` is empty. Something like this would be useful for developing the GUI while we work out the details here.

**Comment by jtarricone** [comment]
Either an option or a separate endpoint, but either way that's a good call, it's pretty maddening to test even w/o throwing errors.

File: src/mmw/apps/modeling/tasks.py [hunk 1]
```diff
@@ -332,13 +332,7 @@ def change_key(modification):
 
 @shared_task
 def run_gwlfe(model_input):
-    # TODO pass real input to model instead of reading it from gms file
-    gms_filename = join(dirname(abspath(__file__)),
-                        'mapshed/data/sample_input.gms')
```
Comment (kdeloach) [inline src/mmw/apps/modeling/tasks.py]: Could we delete this file, if it is no longer needed?

**Comment by rajadain** [comment]
In fact, instead of adding another sample endpoint, we can probably just hold off on merging this until we're ready.

File: src/mmw/apps/modeling/tasks.py (hunk 1 above)
Comment (rajadain) [inline src/mmw/apps/modeling/tasks.py]: Agreed, we should delete that file as part of this PR. But let's not merge until we're at a ready state.

Comment (jtarricone) [inline src/mmw/apps/modeling/tasks.py]: Deleted, thanks for catching that!

**Comment by kdeloach** [comment]
I'm working on a solution for this.

**Comment by rajadain** [comment]
Closing in favor of #1326

**Pull request closed without merging**